src/Classes/IMapEventListener.h

```diff
@@ -6,9 +6,13 @@
 namespace tsg {
 namespace map {
 class IMapEventListener {
-public:
+ public:
   virtual void onMapLoad(cocos2d::TMXTiledMap *) = 0;
-  virtual void onViewCoordinatesChanged(cocos2d::Vec2) =0;
+  virtual void onViewCoordinatesChanged(cocos2d::Vec2) = 0;
+  virtual void onNightTime() = 0;
+  virtual void onDayTime() = 0;
+  virtual void onGameHourPassed() = 0;
```

Comment:
can we use "present simple" time in method names? For example:
Instead of "present perfect" onUpdated use "present simple" onUpdate

This convention helps to improve readability though